### PR TITLE
Working together on code cleanup

### PR DESCRIPTION
**Pull request by ewquon**

### CODE REVIEW

File: Source/EOS.H [hunk 1]
```diff
@@ -17,6 +17,6 @@ amrex::Real getPgivenRTh(amrex::Real density, amrex::Real theta)
     // diagnostic relation for the full pressure (dry air + water vapor) (arw_v4, Eq. 2.16)
     //   density: density of _dry_ air
     //   theta: _moist_ potential temperature
-    return std::pow(density * R_d * theta / p_0, Gamma) ;
+    return p_0 * std::pow(density * R_d * theta / p_0, Gamma);
```
Comment (ewquon) [inline Source/EOS.H:19]: @pkjha-aero can you double-check this? Is there something I'm missing?

**Review by ewquon — COMMENTED** [review]
Compare with current ERF equations rather than WRF doc

File: Source/RK3/RK3.H [hunk 2]
```diff
@@ -69,8 +73,8 @@ void CalcDiffFlux(const MultiFab& cons_in,
                   const amrex::Geometry geom,
                   const amrex::Real* dx, const amrex::Real dt);
 
-void conservedToPrimitive(MultiFab& prim_in, const MultiFab& cons_in);
+//void conservedToPrimitive(MultiFab& prim_in, const MultiFab& cons_in);
 
-void primitiveToConserved(const MultiFab& prim, MultiFab& cons);
+//void primitiveToConserved(const MultiFab& prim, MultiFab& cons);
```
Comment (pkjha-aero) [inline Source/RK3/RK3.H]: Are these remnants from PeleC? Looks like these are never defined.

File: Source/Constants.H [hunk 3]
```diff
@@ -10,9 +10,9 @@
 #define SMALL_E 1.0e-200
 #define SMALLU 1.0e-12
 
-#define R_d     287.06
-#define c_p    1004.5
-#define p_0    1000.0 // Check if is still good for ERF. This value came from PeleC
+#define R_d     287.0 // gas constant for dry air [J/(kg-K)]
+#define c_p    1004.5 // specific heat for at constant pressure for dry air [J/(kg-K)]
+#define p_0    1.0e5 // reference surface pressure [Pa]
```
Comment (pkjha-aero) [inline Source/Constants.H:15]: Updated documentation as well [here](https://github.com/erf-model/ERF/pull/16/commits/e104e6dae6957f0badb62e04ca9820b943fcb74d).

File: Source/EOS.H (hunk 1 above)
Comment (pkjha-aero) [reply Source/EOS.H:19]: I have verified. Your edits match with the literature/ our document https://erf.readthedocs.io/en/latest/Euler_Discretization.html#diagnostic-variables

File: Source/Constants.H [hunk 4]
```diff
@@ -15,7 +15,7 @@
 #define p_0    1.0e5 // reference surface pressure [Pa]
 #define Gamma  1.4
 
-#define CONST_GRAV 9.8
+#define CONST_GRAV 9.81
```
Comment (pkjha-aero) [inline Source/Constants.H:18]: Doc changed [here](https://github.com/erf-model/ERF/pull/16/commits/e9aacd9000e088bc0f6ec8999b729827101f46a5).

File: Source/RK3/RK3.H (hunk 2 above)
Comment (ewquon) [reply Source/RK3/RK3.H]: Yeah, that's why I commented it out. Should we just get rid of them?

Comment (pkjha-aero) [reply Source/RK3/RK3.H]: I think so

**Pull request merged**